mettle/src/stdapi/ui/osx_desktop.m

```diff
@@ -16,7 +16,7 @@
     NSDictionary *properties = [NSDictionary dictionaryWithObjectsAndKeys:
                                 @(compression), kCGImageDestinationLossyCompressionQuality,
                                 nil];
-    CGImageDestinationAddImage(destination, image, (CFDictionaryRef)properties);
+    CGImageDestinationAddImage(destination, image, (__bridge CFDictionaryRef)properties);
```

Comment:
The compiler error actually recommended it, looks like an older compiler requirement.